From b07decbf3a192a51257351a7d6fd6ff7745e2550 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Mon, 21 May 2018 18:59:07 +0530
Subject: [PATCH] libintl.h: Fix detection with AM_GNU_GETTEXT

Without this, the macro tries to use other, more invasive internal
implementation details of gettext to try and detect it and still fails
---
 libintl.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/libintl.h b/libintl.h
index 1fc05c9..0bd6aeb 100644
--- a/libintl.h
+++ b/libintl.h
@@ -26,6 +26,13 @@
 # define LC_MESSAGES 1729       /* Use same value as in GNU gettext */
 #endif
 
+/* Define this to make configure checks successfully find gettext in libintl.
+ * This will cause the AM_GNU_GETTEXT([external]) macro to successfully detect
+ * that gettext is in -lintl, not in libc, and needs #include <libintl.h> */
+#ifndef __GNU_GETTEXT_SUPPORTED_REVISION
+# define __GNU_GETTEXT_SUPPORTED_REVISION
+#endif
+
 #if __GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ > 4)
 # define PROXY_LIBINTL_GNUC_FORMAT(arg_idx) __attribute__((__format_arg__(arg_idx)))
 #else
-- 
2.17.0

